examples/autoqasm/6_Customize_gate_calibrations.ipynb

```diff
@@ -0,0 +1,404 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "dda432a3",
+   "metadata": {},
+   "source": [
+    "# Customize gate calibrations\n",
+    "\n",
+    "Quantum hardware researches frequently require knowledge of native gate implementations on a specific devices, in the form of pulse sequences. The pulse implementation of gates is called \"gate calibrations\". In Amazon Braket, you have access to native gate calibrations provided by quantum hardware providers. Gaining access to gate calibrations empowers researchers to experiment with quantum gate designs and execute quantum programs with customized, improved gate calibrations."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "id": "30fd0b59",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "\n",
+    "import braket.experimental.autoqasm as aq\n",
+    "from braket.experimental.autoqasm import pulse\n",
+    "from braket.experimental.autoqasm.instructions import x, h, y, z, rx, rz, cz, measure\n",
+    "\n",
+    "from braket.aws import AwsDevice\n",
+    "from braket.devices import Devices\n",
+    "from braket.circuits import Gate, QubitSet, FreeParameter\n",
+    "from braket.pulse import DragGaussianWaveform\n",
+    "\n",
+    "\n",
+    "# Rigetti Aspen-M-3 device is the targeted device for this example notebook\n",
+    "device = AwsDevice(Devices.Rigetti.AspenM3)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f9fdfd5e",
+   "metadata": {},
+   "source": [
+    "## Gate calibrations from hardware providers"
```

Comment:
This is not related to AutoQASM, is there redundant documentation elsewhere?

Reply:
Viewing the provider's calibrations is often the only practical way to start. I added some motivation for viewing provider's calibrations

Reply:
Looks good -- so this documentation isn't available to users? It could be helpful if we add just this section to the main branch of the BDK (or wherever such documentation should go) now that native gates has been released